src/clusterfuzz/_internal/system/environment.py

```diff
@@ -705,12 +705,6 @@ def is_posix():
   return os.name == 'posix'
 
 
-def is_trusted_host(ensure_connected=True):
```

Comment:
There is also a test file for this, environment_test.py. It should also be changed, right?

Reply:
The same goes for UntrustedRunnerIntegrationTest